Tasks/AzureFunction/task.json

```diff
@@ -56,17 +56,17 @@
 				"TRACE": "TRACE",
 				"PATCH": "PATCH"
 			},
-			"helpMarkDown": "Select the http method with which the funtion should be invoked with."
+			"helpMarkDown": "Select the http method with which the function should be invoked with."
 		}, {
 			"name": "headers",
 			"type": "multiLine",
 			"label": "Headers",
 			"required": "false",
-			"defaultValue": "{\n\"Content-Type\":\"application/json\"\n}",
+			"defaultValue": "{\n\"Content-Type\":\"application/json\", \n\"PlanUrl\": \"$(system.CollectionUri)\", \n\"ProjectId\": \"$(system.TeamProjectId)\", \n\"HubName\": \"$(system.HostType)\", \n\"PlanId\": \"$(system.PlanId)\", \n\"JobId\": \"$(system.JobId)\", \n\"TimelineId\": \"$(system.TimelineId)\", \n\"TaskInstanceId\": \"$(system.TaskInstanceId)\", \n\"AuthToken\": \"$(system.AccessToken)\"\n}",
```

Comment:
should we put all these values under a single custom header, as it will be easier for the user to access all this information then.